relnotes/ipool.feature.md

```diff
@@ -0,0 +1,7 @@
+* `ocean.util.container.pool.model.IPool`, `ocean.util.container.pool.model.IAggregatePool`
+
+  A new abstract function `isBusy()` has been added to the API of IPool. Classes
+  wishing to inherit from `IPool` must provide an implementation for it.
```

Comment:
Yes, it should target v3.x.x and the milestone would be v4.0.0. I wonder if this can be done differently (see my other comment), so we avoid breaking change.